src/views/task-list-page/task-list-filters/__tests__/task-list-filters.test.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import { render, screen } from '@/test-utils/rtl';
+
+import TaskListFilters from '../task-list-filters';
+
+jest.mock('@/components/page-filters/page-filters', () =>
+  jest.fn(() => <div>Mock page filters</div>)
+);
+
+describe(TaskListFilters.name, () => {
+  it('renders without errors', () => {
```

Comment:
mocking defeats the purpose of rendering without errors.
I see two possible test cases:
- props passed correctly with mocked `page-filters`
- component renders correctly with the original component

Reply:
I agree, but in that case we're essentially just writing the same unit test that page-filters.test.tsx already has, right?
If at all, it would be better to make assertions on the config and not the component

Reply:
yes, but this is said with the perspective that the we only have `page-filters` in the component. Not that we are rendering the whole filters regardless of the content.

The above also will be covered with the props assertion too.

Reply:
Got it, I've just done a plain render without any extra assertions.